recipes/pipefunc/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pipefunc" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pipefunc-{{ version }}.tar.gz
+  sha256: 988ef01177d73cfd8f38c86999ac4819818c562e75aee101b34fc6502877b623
+
+build:
+  skip: true  # [py<38]
```

I believe this one can be noarch.

```suggestion
  noarch: python
```